server/samples/clients/python/README.md

```diff
@@ -0,0 +1,37 @@
+# gRPC Client (Python) for Asherah Server
+A simple client application that demonstrates integrating with Asherah Server via a generated gRPC client.
+
+## Running the client
+Ensure the Asherah Server is running locally and listening on `unix:///tmp/appencryption.sock` and run:
```

Comment:
Should we add small sub-sections about running client in different modes - once, continuous, multiple clients ?